members/H001042.yaml

```diff
@@ -3,131 +3,73 @@ contact_form:
   method: POST
   action: ""
   steps:
-    - visit: "http://www.hirono.senate.gov/contact/email"
+    - visit: "https://www.hirono.senate.gov/help/email"
     - fill_in:
-      - name: fname
-        selector: "#fname"
+      - name: firstname
+        selector: input#input-B03E05C4-4040-F985-52CD-9CA44623DE5E
         value: $NAME_FIRST
         required: true
-      - name: lname
-        selector: "#lname"
+      - name: lastname
+        selector: input#input-B03E04BD-4040-F985-52CD-48E090E769DA
         value: $NAME_LAST
         required: true
-      - name: mailing_streetAddress1
-        selector: "#mailing_streetAddress1"
+      - name: street
+        selector: input#input-B03E05F1-4040-F985-52CD-A580E72A7D41
         value: $ADDRESS_STREET
         required: true
-      - name: mailing_streetAddress2
-        selector: "#mailing_streetAddress2"
+      - name: street
```

Comment:
might be helpful to name this street2 for clarity

Reply:
done!